react-native-git-upgrade/cli.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+require('babel-register')({
+  presets: [
+    require('babel-preset-es2015-node'),
+    require('babel-preset-stage-3')
+  ],
+  only: /(react-native-git-upgrade\/(?!(node_modules)))|(local-cli\/generator)/,
```

Comment:
👍 The value means "all files inside the `react-native-git-upgrade` folder except its `node_modules` subdirectory AND the `local-cli/generator` folder"

I've basically added the `local-cli/generator` folder because I encountered syntax errors in it.  

A closer look reveals that the new "copyAndReplace" generator uses trailing comma in function calls [here](https://github.com/facebook/react-native/blob/master/local-cli/generator/copyProjectTemplateAndReplace.js#L52) and [here](https://github.com/facebook/react-native/blob/master/local-cli/generator/copyProjectTemplateAndReplace.js#L63).
If I remove them it works without transpiling the generator, but I guess it depends of the Node version (I use Node 7), an older version of Node might fail somewhere else...

I think we should be consistent: if the generator is transpiled with `react-native upgrade`, it must be the same with `react-native-git-upgrade`.